index.html

```diff
@@ -166,9 +175,9 @@ <h2 class="card-title">Multi-post stories</h2>
             <div class="card-item">
                 <h2 class="card-title test">About Myself</h2>
                    
-                    <p class="primary-text">
-                        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-                    </p>
+                <p class="primary-text">
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>    
                     <div class="letsConnect">
                         <h2> Let's Connect</h2>
                         <ul class="social-item">
```

Comment:
- [ ]  Your design should be similar to the one from Figma design, please fix the space between the icons 
![image](https://user-images.githubusercontent.com/43877689/177986914-3a350525-6d90-4b30-8e3c-e5b8ce90374f.png)